### PR TITLE
fix(appeals/web): fixes appellant case rendering

### PR DESCRIPTION
**Pull request by fabfab-kc**
Fixes the current error rendering the appellant case, result of service user restructuring.

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/__snapshots__/appellant-case.test.js.snap [hunk 1]
```diff
@@ -316,7 +316,7 @@ exports[`appellant-case GET /appellant-case with unchecked documents should rend
         <div class="govuk-summary-card__content">
             <dl class="govuk-summary-list">
                 <div class="govuk-summary-list__row"><dt class="govuk-summary-list__key"> Appellant name</dt>
-                    <dd class="govuk-summary-list__value">Fiona Burgess</dd>
+                    <dd class="govuk-summary-list__value"></dd>
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/__snapshots__/appellant-case.test.js.snap:319]: This looks like the test data just needs updating to reflect the change (in `appeals/web/testing/app/fixtures/referencedata.js`)

**Review by james-hill-kc — COMMENTED** [review]
Looks good! Just one comment from me

File: appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/__snapshots__/appellant-case.test.js.snap (hunk 1 above)
Comment (fabfab-kc) [reply appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/__snapshots__/appellant-case.test.js.snap]: Well spotted! Added another commit...

**Review by james-hill-kc — APPROVED** [review]
LGTM!

**Pull request merged**